lib/ramble/ramble/application.py

```diff
@@ -126,6 +129,7 @@ def copy(self):
         new_copy.set_variables(self.variables.copy(), self.experiment_set)
         new_copy.set_internals(self.internals.copy())
         new_copy.set_template(False)
+        new_copy.set_repeats((False, 0))
```

Comment:
This copy method is used for creating the copies of an experiment that get inserted as chained experiments, so repeats should be set to 'off'. I've changed this now to have it run 'repeats.set_repeats()' with no args, which uses the default values of (False, 0), so that it's not hardcoded in multiple places